pkg/solana/client/client.go

```diff
@@ -66,10 +67,45 @@ type Client struct {
 
 	// provides a duplicate function call suppression mechanism
 	requestGroup *singleflight.Group
+
+	// MultiNode
+	pollInterval               time.Duration
```

Comment:
We should not modify existing clients. MultiNode feature flag should guarantee that if it's disabled we'll run pre-MultiNode code

Reply:
Created `MultiNodeClient` which will wrap `Client` with the additional requirements for MultiNode